Playlist_Genre_Jazz.php

```diff
@@ -2,7 +2,7 @@
 <html lang = "eng">
```

Comment:
all changes are the same in 'Playlist_Genre_Jazz.php' just different queries and titles in V2

Reply:
also change in spacings + breaks